src/Users/README.md

```diff
@@ -123,6 +180,82 @@ if (!$result) {
 }
 ```
 
+### DELETE OWN ACCOUNT
+```php
+$result = \ATDev\RocketChat\Users\User::deleteOwnAccount(hash("sha256","[USER PASSWORD]"));
+if (!$result) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### DEACTIVATE IDLE
+```php
+$count = \ATDev\RocketChat\Users\User::deactivateIdle(5, "guest");
+if (!$count) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### PRESENCE
+```php
+$presence = \ATDev\RocketChat\Users\User::presence();
+if (!$presence) {
+	$error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### GET PRESENCE
+```php
+$user = new \ATDev\RocketChat\Users\User("[USER ID]");
+// or by username
+$user = (new \ATDev\RocketChat\Users\User())->setUsername("[USERNAME]");
+$userPresence = \ATDev\RocketChat\Users\User::getPresence($user);
```

Comment:
This is incorrect usage. We should call instance method, since we get instance presence.